website/src/blogs/provisioning-openebs-jiva-volumes-via-csi.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Provisioning OpenEBS Jiva volumes via CSI
+author: Prateek Pandey
+author_info: Contributor and Maintainer @OpenEBS. Software Developer at @mayadata_inc. Open Source Enthusiast
+date: 09-06-2021
+tags: CSI, OpenEBS, Jiva
+excerpt: Container Storage Interface (CSI), is the new model for integrating storage system drivers with container orchestration systems like Kubernetes. This new interface is a major benefit to the container ecosystem as it standardizes the model of integrating storage systems with container orchestration systems.
+not_has_feature_image: false
```

Comment:
since we have feature images, we don't need to use defaultImage

![Screenshot 2021-09-17 at 2 24 24 PM](https://user-images.githubusercontent.com/6524419/133754779-4e211192-a68d-45cc-8ba7-f9b8a22ed2fb.png)

Reply:
Yes, that is why not_has_feature_image is set to false. So it means it has a feature image

Reply:
separate PR for fixing logic

![Screenshot 2021-09-17 at 2 58 26 PM](https://user-images.githubusercontent.com/6524419/133762762-3414a6b0-4fe3-4199-9b45-04a55cae255c.png)

Reply:
Yes. Will fix the logic in separate PR